generators/pythonv2/codegen/src/ast/Reference.ts

```diff
@@ -16,6 +16,8 @@ export declare namespace Reference {
         genericTypes?: Type[];
         /* The alias of the reference */
         alias?: string;
+        /* The path to the attribute of the reference */
+        attrPath?: string[];
```

Comment:
Open to feedback on naming for this.

Reply:
would probably name it attribute and type as `string[]` like you have it